src/main/java/uk/gov/hmcts/darts/task/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -132,6 +132,7 @@ public void reloadTaskByName(String taskName) {
         }
         boolean found = false;
         for (AutoloadingAutomatedTask autoloadingAutomatedTask : autoloadingAutomatedTasks) {
+            System.out.println("TMP: " + autoloadingAutomatedTask.getAutomatedTaskName() + " -- " + automatedTaskName);
```

Comment:
I take it you are using system.out instead of logging here because logging is not yet setup? if so, might be worth adding a comment so someone later doesn't come along and think its a mistake and change it to logging

Reply:
I have removed this as this was left over from some dev testing :)